runtime/logging/common/src/aws/smithy/kotlin/runtime/logging/LoggerExt.kt

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package aws.smithy.kotlin.runtime.logging
-
-/**
- * Logger that wraps [inner] and adds key/value pairs from the log context to every
- * logging message. This is a (very) rough MDC/structured logging abstraction
- */
-private class ContextualLogger(
```

Comment:
nice, I never loved this. It was a half baked attempt at a structured logging context.